README.md

```diff
@@ -66,9 +66,8 @@ Logging
 
 Web Applications
 ----------------
-* [ ] Must have a CSP with
-  * [ ] a report-uri pointing to the service's own `/__cspreport__` endpoint
```

Comment:
Should we keep it as a recommendation? Something along the lines of `developers who may want visibility into CSP violations could set a report-uri endpoint, but this is not a security requirement`

Reply:
Probably worth mentioning in some form.

There's a larger question of: does this checklist just represent requirements or does it also include suggested ways of satisfying those requirements and other recommendations?

Should we adopt IETF RFC style MUST, MAY, etc. definitions? Currently I think everything is a MUST but then there are suggestions for dep management tools and stuff (which might be MAYs).

Reply:
This is probably the wrong way of going about it, but I actually like the gray area, because it often triggers good conversations with devs.